### PR TITLE
Tada68 Fixes and Minor Refactoring

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

A recent PR broke the layout macro references, which broke all the keymaps. Fixed that, and saw some issues and things that could be improved, so figured "Eh, I'm in here already."

Also adds 65_iso Community Layout support

## Types of Changes

- [x] Bugfix
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
CI error on `laas` keymap, but it seems to have been broken prior to the commit that broke the codebase.

File: keyboards/tada68/tada68.h [hunk 1]
```diff
@@ -103,5 +104,3 @@
 
 void matrix_init_user(void);
```
Comment (fauxpark) [inline keyboards/tada68/tada68.h]:
```suggestion
```

File: keyboards/tada68/tada68.h [hunk 2]
```diff
@@ -103,5 +104,3 @@
...
 void matrix_scan_user(void);
```
Comment (fauxpark) [inline keyboards/tada68/tada68.h]:
```suggestion
```

File: keyboards/tada68/tada68.h [hunk 3]
```diff
@@ -55,21 +54,23 @@
...
 { \
     {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e}, \
-    {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k2c, k1e}, \
+    {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, k1e}, \
     {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, XXX, k2d, k2e}, \
     {k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k3e}, \
     {k40, k41, k42, XXX, XXX, k45, XXX, XXX, XXX, k49, k4a, k4b, k4c, k4d, k4e}  \
 }
 
+#define LAYOUT_iso LAYOUT_65_iso
```
Comment (zvecr) [inline keyboards/tada68/tada68.h]: Dont we also need backward compat for `LAYOUT_ansi` given it was also changed?
```suggestion
#define LAYOUT_iso LAYOUT_65_iso
#define LAYOUT_ansi LAYOUT_65_ansi
```

Comment (fauxpark) [reply keyboards/tada68/tada68.h]: It seems to have already been there, so it won't show up in the diff. It's on line 39 (40 master)

Comment (noroadsleft) [reply keyboards/tada68/tada68.h]: :point_up_2:

**Pull request merged**